src/main/java/io/github/darealturtywurty/turtychemistry/core/init/BlockInit.java

```diff
@@ -129,6 +133,14 @@ public final class BlockInit extends AbstractInit {
     public static final RegistryObject<ClayAlloyFurnaceBlock> CLAY_ALLOY_FURNACE = BLOCKS.register("clay_alloy_furnace",
             () -> new ClayAlloyFurnaceBlock(BlockBehaviour.Properties.copy(Blocks.TERRACOTTA).dynamicShape().noOcclusion()));
 
+    public static final RegistryObject<RubberTreeBlock> RUBBER_TREE_BLOCK = register("rubber_tree_block",
+            () -> new RubberTreeBlock(BlockBehaviour.Properties.of(Material.WOOD)),new Item.Properties().tab(TurtyChemistry.TAB));
+    public static final RegistryObject<RubberTreeBlockStripped> RUBBER_TREE_BLOCK_STRIPPED = register("rubber_tree_block_stripped",
+            () -> new RubberTreeBlockStripped(BlockBehaviour.Properties.of(Material.WOOD)),new Item.Properties().tab(TurtyChemistry.TAB));
+
+    public static final RegistryObject<RubberTreeLeafBlock> RUBBER_TREE_LEAF_BLOCK = register("rubber_tree_leaf_block",
```

Comment:
I'm not sure why you named it so weirdly, but this would be better
```suggestion
    public static final RegistryObject<RubberTreeLeafBlock> RUBBER_TREE_LEAVES = register("rubber_tree_leaves",
```